projects/prison-custody-status-to-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/integrations/delius/recall/RecallService.kt

```diff
@@ -132,7 +163,7 @@ class RecallService(
                 notes = "Reason for Recall: ${recallReason.description}",
                 staffId = orderManager.staffId,
                 teamId = orderManager.teamId,
-                createdDatetime = recall.createdDatetime,
+                createdDatetime = recall?.createdDatetime ?: recallDateTime,
```

Comment:
Should this default to `ZonedDateTime.now()` when we're not linking it to a recall?  The contact date is still set to recallDateTime.